daemon/src/maker_cfd.rs

```diff
@@ -1060,6 +960,64 @@ where
     }
 }
 
+#[xtra_productivity]
+impl<O, M, T, W> Actor<O, M, T, W>
+where
+    O: xtra::Handler<oracle::MonitorAttestation>,
+    M: xtra::Handler<monitor::StartMonitoring>,
+    W: xtra::Handler<wallet::TryBroadcastTransaction>,
+{
+    async fn handle_setup_completed(&mut self, msg: setup_maker::Completed) -> Result<()> {
```

Comment:
There is a lot that can go wrong here, please use `log_error` for the time being until we have something better.

Note that you are also dropping this error within `setup_maker::Actor` so it won't show up anywhere.